README.md

```diff
@@ -26,339 +26,213 @@
-application.
-
-The current release of the Triton Inference Server is 2.21.0 and
-corresponds to the 22.04 release of the tritonserver container on
-[NVIDIA GPU Cloud (NGC)](https://ngc.nvidia.com). The branch for this
-release is
-[r22.04](https://github.com/triton-inference-server/server/tree/r22.04).
-
-## Features
-
-* [Deep learning
-  frameworks](https://github.com/triton-inference-server/backend).
-  Triton supports TensorRT, TensorFlow GraphDef, TensorFlow
-  SavedModel, ONNX, PyTorch TorchScript and OpenVINO model
-  formats. Both TensorFlow 1.x and TensorFlow 2.x are
-  supported. Triton also supports TensorFlow-TensorRT, ONNX-TensorRT
-  and PyTorch-TensorRT integrated models.
-
-* [Machine learning
-  frameworks](https://github.com/triton-inference-server/fil_backend).
-  Triton supports popular machine learning frameworks such as XGBoost,
-  LightGBM, Scikit-Learn and cuML using the [RAPIDS Forest Inference
-  Library](https://medium.com/rapids-ai/rapids-forest-inference-library-prediction-at-100-million-rows-per-second-19558890bc35).
-
-* [Concurrent model
-  execution](docs/architecture.md#concurrent-model-execution). Triton
-  can simultaneously run multiple models (or multiple instances of the
-  same model) using the same or different deep-learning and
-  machine-learning frameworks.
-
-* [Dynamic batching](docs/architecture.md#models-and-schedulers). For
-  models that support batching, Triton implements multiple scheduling
-  and batching algorithms that combine individual inference requests
-  together to improve inference throughput. These scheduling and
-  batching decisions are transparent to the client requesting
-  inference.
-
-* [Extensible
-  backends](https://github.com/triton-inference-server/backend). In
-  addition to deep-learning frameworks, Triton provides a *backend
-  API* that allows Triton to be extended with any model execution
-  logic implemented in
-  [Python](https://github.com/triton-inference-server/python_backend)
-  or
-  [C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api),
-  while still benefiting from full CPU and GPU support, concurrent
-  execution, dynamic batching and other features provided by Triton.
-
-* Model pipelines using
+under-development progress towards the next release. The current release is 
+version [2.21.0](https://github.com/triton-inference-server/server/tree/r22.04)
+and corresponds to the 22.04 container release on 
+[NVIDIA GPU Cloud (NGC)](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tritonserver).**
+
+----
+Triton Inference Server is an inference serving software that streamlines AI 
+inferencing. Triton enables teams to deploy any AI model from multiple deep 
+learning and machine learning frameworks, including TensorRT, TensorFlow, PyTorch, 
+ONNX, OpenVINO, Python and more. Triton supports inference across cloud, data center,
+edge and embedded devices on NVIDIA GPUs, x86 and ARM CPU, or AWS Inferentia. 
```

Comment:
Are there links to the other supported devices, or just call out inferentia? Also note, most of these links are farther down. This is just a statement that marketing uses. I can add it in the end near Jetson support.

Reply:
Most of the backends support Nvidia GPUs and x86/ARM CPUs. 
To run on AWS inferentia we have to specifically use AWS Inferentia python script within python backend as described in the above link. It is important to call out the distinction.

I don't think we have hardware supported documentation yet. @GuanLuo  Let me know if you are aware of any.

>This is just a statement that marketing uses. I can add it in the end near Jetson support.

Given this case I think our best choice right now is add a footnote to AWS Inferentia, given its special support.
The footnote can say:
`The support for AWS Inferentia is only available through python backend using AWS Neuron python module. See [here](https://github.com/triton-inference-server/python_backend/tree/main/inferentia) for more details.`

This will prevent a lot of misinterpretations. This is how a footnote will look: https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/

Reply:
I don't think we have this kind of support matrix now. As @tanmayv25 mentioned we only have limited support for running AWS Inferentia, so some clarification may be needed.